### PR TITLE
Help button in admin

### LINKED ISSUE #1449: Add links to the  help pages on Admin pages of the site.

### PR DESCRIPTION
**Pull request by actlikewill**
- adds help button in admin
image

closes #1449

### CODE REVIEW

File: peachjam/templates/admin/change_form.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{% extends "admin/change_form.html" %}
+{% block extra_actions %}
+  {% if PEACHJAM_SETTINGS.help_link %}
+    <a href="{{ PEACHJAM_SETTINGS.help_link }}{{ adminform.model_admin.help_topic }}"
+       class="btn btn-block btn-info">Help</a>
+  {% endif %}
```
Comment (longhotsummer) [inline peachjam/templates/admin/change_form.html:6]: `Help` must be translateable.

Comment (actlikewill) [reply peachjam/templates/admin/change_form.html:7]: Added

File: peachjam/templates/admin/change_form.html [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{% extends "admin/change_form.html" %}
+{% block extra_actions %}
+  {% if PEACHJAM_SETTINGS.help_link %}
```
[the remark this answers is not in the document]
Comment (actlikewill) [reply peachjam/templates/admin/change_form.html]: Added

**Comment by longhotsummer** [comment]
Looks great. 

Please can you see if it's possible to add a similar button here, so that there's a help button in both the listing and detail views.
image

Please also add links for the docs on Users, Registries and Matter Types: https://liieditors.docs.laws.africa/site-admin/add-court-registries

image

**Review by longhotsummer — APPROVED** [review]

**Pull request merged**